code/game/machinery/adv_med.dm

```diff
@@ -483,11 +489,13 @@
 					infection = "Acute Infection:"
 				if(INFECTION_LEVEL_TWO + 200 to INFECTION_LEVEL_TWO + 300)
 					infection = "Acute Infection+:"
-				if(INFECTION_LEVEL_TWO + 300 to INFINITY)
+				if(INFECTION_LEVEL_TWO + 300 to INFECTION_LEVEL_TWO + 400)
 					infection = "Acute Infection++:"
+				if(INFECTION_LEVEL_THREE to INFINITY)
+					infection = "Septic:"
```

Comment:
It turns out this is actually a problem. You have code to handle 800-900 and 1000-INFINITY but nothing to handle 900-1000. So while germ level is between 900 and 1000, the print out shows nothing about infection.

![image](https://user-images.githubusercontent.com/42044220/76924148-7c255200-68a3-11ea-9e31-f3f47578f7b5.png)

Reply:
Ironically this is how it's always been for limbs, so this means this entire time you could get really unlucky and have 920 infection on your chest and not even be aware of it VIA scanner. Beautiful. I'll figure out how to fix this tomorrow, shouldn't be that hard.